client/schemas/blueprint-spec2-schema.json

```diff
@@ -302,6 +308,43 @@
             },
             "title": "Envrironment Variable"
         },
+        "WorkflowTrigger": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "type":{
+                    "type": "string",
+                    "enum": [
+                        "manual",
+                        "event",
+                        "cron"
+                    ]
+                },
+                "event":{
+                    "type": "string",
+                    "enum": [
+                        "Drift Detected",
+                        "Updates Detected",
+                        "Approval request triggered",
+                        "Environment Ended",
+                        "Environment Launched",
+                        "Environment Active With Error",
+                        "Environment Ending Failed",
+                        "Environment Force Ended",
+                        "Environment Extended",
+                        "Collaborator Added",
+                        "Environment Idle"
```

Comment:
are these values case sensitive?
I would prefer if we can support case insensitivity

also I would change "Approval request triggered "to "Approval Request Triggered", to match the rest of the events